keps/167-metrics-collector/README.md

```diff
@@ -0,0 +1,180 @@
+# KEP-167: Metrics Collector
+
+## Summary
+Define a new Custom Resource `MetricsCollector` that will periodically collect metrics from the endpoint where metrics are published.
+
+`MetricsCollector` allows users to periodically collect arbitrary metrics exposed to them from a specified server.
+This resource is primarily intended to be used in conjunction with the `Scenario` to benchmark simulators, however, it also provides a more general and simple functionality that can be used in other ways. (`Scenario` is the CR to run scenario-based simulation introduced in KEP-140.)
+
+## Motivation
+With `Scenario` introduced in KEP-140, we are now able to experiment with scheduling on a somewhat larger scale.
+And you will surely want metrics against which you can measure the performance of your scheduler.
+This KEP defines a new custom resource `MetricsCollector` to collect and record metrics.
+
+### Goals 
+- Allow users to obtain scheduler metrics periodically.
+- Combined with `Scenario`, it allows users to record metrics during scenario execution.
```

Comment:
It's kinda motivation, not goal itself. Remove it from here and move to somewhere else.